python/GafferUI/FileMenu.py

```diff
@@ -92,8 +92,11 @@ def __open( currentScript, fileName ) :
 	script = Gaffer.ScriptNode()
 	script["fileName"].setValue( fileName )
 
-	messageWidget = GafferUI.MessageWidget()
-	with messageWidget.messageHandler() :
+	with GafferUI.ErrorDialogue.ErrorHandler(
+		title = "Errors Occurred During Loading",
+		closeLabel = "Oy vey",
```

Comment:
"Ok", "Ignore", "Oops"? I don't mind leaving it "Oy Vey" either really, but I wasn't sure if everyone knows what it means...